### DISCUSSION TITLE
你好，MouseInc恢复、重新打开刚才关闭的网页的鼠标手势是什么呀？

### DISCUSSION THREAD

**Discussion by cc19264997** [shuax/MouseInc.Settings#260] [Q&A] [answered]
你好，MouseInc恢复、重新打开刚才关闭的网页的鼠标手势是什么呀？没看到啊   每次都需要ctrl+shift+T

**Comment by shuax** [comment, marked as answer]
你可以自己加一个手势

**Comment by Eunknight** [comment]
鼠标手势 -> 特定动作 -> 浏览器 -> 新建
选择手势：_  ->  名称：重新打开关闭的标签页  ->  动作：Raw (复制粘贴以下配置)

> [
> 	["SendKeys", "Ctrl+Shift+T"]
> ]

确定

保存

**Reply by cc19264997** [reply]
> 鼠标手势 -> 特定动作 -> 浏览器 -> 新建 选择手势：_ -> 名称：重新打开关闭的标签页 -> 动作：Raw (复制粘贴以下配置)
> 
> > [
> > ["SendKeys", "Ctrl+Shift+T"]
> > ]
> 
> 确定
> 
> 保存

已经搞好了，还是谢谢你
